packages/perseus-editor/src/styles/perseus-editor.less

```diff
@@ -279,15 +279,6 @@
 
     // Question area styles
     #problemarea {
-        /* Override the min-height property used for displaying the problem.
```

Comment:
I _think_ that once we're fully on `ServerItemRenderer` we won't have a `problemarea` anywhere anymore.  Looking around in this repo, I see it being rendered by the `item-editor.tsx` and nowhere else.